### DISCUSSION TITLE
how to add font in v.7

### DISCUSSION THREAD

**Discussion by goodlythink** [mantinedev/mantine#6411] [Q&A]
How to add external font in mantine v.7
![2024-06-19_14-28-06](https://github.com/mantinedev/mantine/assets/12431734/8cc19a4d-d019-46fd-a193-a2a9f692ca52)

**Comment by rtivital** [comment]
- https://help.mantine.dev/q/next-load-fonts
- https://help.mantine.dev/q/vite-load-fonts
